src/callbacks/alive.jl

```diff
@@ -15,7 +15,9 @@ end
 
 function AliveCallback(; analysis_interval=0,
                          alive_interval=analysis_interval÷10)
-  condition = (u, t, integrator) -> alive_interval > 0 && ((integrator.iter % alive_interval == 0 && (analysis_interval == 0 || integrator.iter % analysis_interval != 0)) || t == integrator.sol.prob.tspan[2])
+  condition = (u, t, integrator) -> alive_interval > 0 && (
+    (integrator.iter % alive_interval == 0 && (analysis_interval == 0 || integrator.iter % analysis_interval != 0)) ||
+    t == integrator.sol.prob.tspan[2] || isempty(integrator.opts.tstops))
```

Comment:
What does `isempty(integrator.opts.tstops)` check for? Is this non-empty if another callback marked this as the last time step and that the solver finishes afterwards?